examples/gke/gke_castai_iam/versions.tf

```
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 0.13"
+
+  required_providers {
+    castai = {
+      source  = "castai/castai"
+      version = ">= 3.11.0"
```

nitpic: consider updating to:
```suggestion
      version = ">= 5.1.0"
```